db/health/migrate/20220407204625_add_open_status_to_vprs.rb

```diff
@@ -0,0 +1,5 @@
+class AddOpenStatusToVprs < ActiveRecord::Migration[6.1]
+  def change
+    add_column :health_flexible_service_vprs, :open, :boolean, default: :true
```

Comment:
Is this going to open a bunch of existing VPRS?

Reply:
Yes, but since Mike gets all the VPRs right now, and they don't have a notion of open/closed (or the follow ups that will close them), I was thinking that it made more sense to show all of them as open, and either Mike can file follow ups to close the ones that are no longer active, or he can tell us which ones to close.